library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -142,11 +135,6 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         ).mapNotNull {
```

Comment:
Should be an easy rip out looks like we weren't even exposing direction in the bindings anyways. @neekolas

Reply:
Wait. This is just querying from the local DB. You can still do that in any order you want.

I only care about removing the ability to query from the network in descending order

Reply:
Oooo... 🤦‍♀️ Thats all entirely in libxmtp for V3! So no SDK issue here. I'll close this.